contracts/tests/MACI.test.ts

```diff
@@ -259,6 +259,39 @@ describe("MACI", function test() {
       ]);
       maciState.polls.get(pollId)?.publishMessage(message, padKey);
     });
+
+    it("should allow to deploy a new poll even before the first one is completed", async () => {
+      const tx = await maciContract.deployPoll(
+        duration,
+        treeDepths,
+        coordinator.pubKey.asContractParam() as { x: BigNumberish; y: BigNumberish },
+        verifierContract,
+        vkRegistryContract,
+        EMode.QV,
+        { gasLimit: 10000000 },
+      );
+      const receipt = await tx.wait();
+      expect(receipt?.status).to.eq(1);
+      expect(await maciContract.nextPollId()).to.eq(2);
+    });
+
+    it("should allow any user to deploy a poll", async () => {
```

Comment:
I'm just curious that why we wanna allow any user to deploy poll?

Reply:
actually good question, I removed the modifier as now each poll is independent from each other and can run concurrently, so it doesn't really matter who deploys it (+ we save a tiny bit of gas), it will have its own set of contracts deployed and their own coordinator key. Anyone could now re use the same signups (so users which passed a certain anti-sybil requirements) and run their poll for these users 
Do you think there's a reason why we should keep it `onlyOwner`?

Reply:
No I'm just curious why you consider to make this change, also might change little bit to the UI display of maci-rpgf design. Agree with that.

Reply:
I could see an argument where it might be best to keep it callable only by the owner, if for instance you don't want other users from deploying polls from your contract, and filling up you state tree (assuming the gatekeeping requirements are not very custom, like for instance proof of humanhood vs owning a NFT)

Reply:
that's also a good way, but would need to add an argument here and affecting a lot of codes calling this function 🤔 not sure if it's worth doing this.